### PR TITLE
updated README language

### PR DESCRIPTION
**Pull request by drewbennison**
# Why Is This Happening
Updated README for purposes of testing process.

# Files Changed
`README.md`
- Updated language.

# Test Results
Untested.

# Breaking Changes and Comments
N/A

### CODE REVIEW

**Comment by drewbennison** [comment]
@kylebennison first PR with discussion.

File: README.md [hunk 1]
```diff
@@ -9,4 +9,4 @@ To contribute, [pick an issue or create a new issue](https://github.com/kylebenn
 
 When your work is ready to be reviewed, open a Pull Request to merge your branch into dev and fill out the PR template.
 
-You will need to [install the statRdaysCFB package](https://github.com/kylebennison/statRdaysCFB) in order to access data and run most scripts. You will also need a CFBdata [API key](https://collegefootballdata.com/key).
\ No newline at end of file
+You will need to [install the statRdaysCFB package](https://github.com/kylebennison/statRdaysCFB) in order to access data and run most scripts. You will also need a free CFBdata [API key](https://collegefootballdata.com/key).
```
Comment (kylebennison) [inline README.md:12]: this is what a comment would look like.

You can add comments in-line in code or generally with your review, and then you can approve, request changes, or just comment.

**Review by kylebennison — APPROVED** [review]
looks good

File: README.md (hunk 1 above)
Comment (kylebennison) [reply README.md:12]: FYI typically comment writers "resolve" their comments after the PR submitter made changes and the commenter is happy with them.

Last person to approve the PR typically merges.

**Pull request merged**